packages/analytics/add-global-link-handler/index.ts

```diff
@@ -94,16 +95,3 @@ export function addGlobalLinkHandler(
 
   hasHandler = true
 }
-
-function getSegmentAnonymousId(): string | null {
-  if (
-    typeof window !== undefined &&
-    window.analytics &&
-    window.analytics.user &&
-    typeof window.analytics.user === 'function'
-  ) {
-    return window.analytics.user().anonymousId()
-  } else {
-    return null
-  }
-}
```

Comment:
I removed this method in favor of `getSegmentId()` in `../analytics-js-helpers.ts`